### PR TITLE
Fix block heights; conditionally flatten inner statement notch.

### PR DESCRIPTION
**Pull request by tmickel**
#388 and #385.

Fixes positioning in the flyout, too :)

### CODE REVIEW

File: core/block_render_svg_vertical.js [hunk 1]
```diff
@@ -507,6 +508,26 @@ Blockly.BlockSvg.prototype.renderCompute_ = function(iconWidth) {
     } else {
       input.renderWidth = 0;
     }
+
+    // If the input is a statement input, determine if a notch
+    // should be drawn at the inner bottom of the C.
+    if (input.connection && input.connection.type === Blockly.NEXT_STATEMENT) {
+      // Walk down linkedBlock's nextConnection to determine if
```
Comment (rachel-fenichel) [inline core/block_render_svg_vertical.js]: This should be a helper function in block.js (maybe getLastBlockInStack()), not implemented inline here.

Looks like the android one is called getLastBlockInSequence()
https://github.com/google/blockly-android/blob/6c4d4376c1390d73d67a13375f72a68dbb341444/blocklylib-core/src/main/java/com/google/blockly/model/Block.java#L568

File: core/block_render_svg_vertical.js [hunk 2]
```diff
@@ -354,6 +354,7 @@ Blockly.BlockSvg.prototype.getHeightWidth = function() {
   if (nextBlock) {
```
Comment (rachel-fenichel) [inline core/block_render_svg_vertical.js:354]: This isn't new, but getHeightWidth is now defined in block_svg.js, block_render_svg_horizontal.js, and block_render_svg_vertical.js.  One of these should be unnecessary.

Comment (tmickel) [inline core/block_render_svg_vertical.js:354]: The one in block_svg should be unnecessary (and overridden).

File: core/block_render_svg_vertical.js (hunk 1 above)
Comment (tmickel) [inline core/block_render_svg_vertical.js]: Sounds good.

File: core/block_render_svg_vertical.js (hunk 2 above)
Comment (tmickel) [inline core/block_render_svg_vertical.js:354]: Removed it.

File: core/block_render_svg_vertical.js (hunk 1 above)
Comment (tmickel) [inline core/block_render_svg_vertical.js]: Turns out there is a similar one `lastConnectionInStack_`

**Comment by tmickel** [comment]
Please take another look :) I wasn't sure whether to rename lastConnectionInStack_ to be public...

**Comment by rachel-fenichel** [comment]
Yes, please rename it to public.  Then lgtm.

**Pull request merged**